code/_onclick/hud/hud.dm

```diff
@@ -265,4 +267,6 @@
 		show_hud(HUD_STYLE_STANDARD,mymob)
 
 /datum/hud/proc/update_locked_slots()
-	return
\ No newline at end of file
+	return
+
+/obj/screen/holomap
```

Comment:
Why is this type empty?

Reply:
it's just a placeholder where a holomap can be drawn